input/resources/StructureDefinition-profile-SignalUSCoreServiceRequest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "signal-profile-us-core-service-request",
+  "url": "https://signalbhn.org/fhir/StructureDefinition/StructureDefinition-profile-SignalUSCoreServiceRequest",
+  "name": "StructureDefinition-profile-SignalUSCoreServiceRequest",
+  "status": "draft",
+  "fhirVersion": "4.0.1",
+  "kind": "resource",
+  "abstract": false,
+  "type": "ServiceRequest",
+  "baseDefinition": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-servicerequest",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "ServiceRequest.status",
+        "path": "ServiceRequest.status",
+        "binding": {
+          "strength": "required",
+          "valueSet": "http://hl7.org/fhir/ValueSet/request-status"
+        }
+      },
+      {
+        "id": "ServiceRequest.intent",
+        "path": "ServiceRequest.intent",
+        "binding": {
+          "strength": "required",
+          "valueSet": "http://hl7.org/fhir/ValueSet/request-intent"
+        }
+      },
+      {
+        "id": "ServiceRequest.category:us-core",
```

Comment:
Change back US Core to it's default definition.  Don't change this ValueSet, define a new slice under ServiceRequest.category, I would suggest `ServiceRequest.category:serviceCategory`

Reply:
@enjoysparkling Please review the changes https://github.com/enjoysparkling/signal-mso-fhir-profiles/pull/84/commits/7ff59fdbf25fb0ab57b0fd22d5a94a14ff3ab57d